sites/fast-component-explorer/package.json

```diff
@@ -97,7 +97,6 @@
     "favicons-webpack-plugin": "0.0.9",
     "focus-visible": "^4.1.5",
     "html-webpack-plugin": "^3.2.0",
-    "husky": "^1.2.0",
```

Comment:
can you also remove `lint-staged` from this and the creator `package.json`?

Reply:
@janechu  @nicholasrice Could you confirm that the lint-staged package and configuration should be removed for all sites?  fast-color-explorer, fast-component-explorer, fast-creator, and site-utilities?

Reply:
hmm - wait why are you removing `lint-staged`?

Reply:
@janechu can you explain why you want lint-staged removed? I'm assuming given the latest response from @nicholasrice and @chrisdholt they don't.  Do you only want it removed from the creator?  Please confirm all.  Thanks

Reply:
doesn't seem like we need it in these package.json files to run since it's left out in the other packages and we still seem to be linting during commits

Reply:
I think we only need this in the root - we should just make sure that the pre-commit hook runs in these package with them removed.

Reply:
Tested and confirmed.  Thanks all.